agent/src/agent/modules/logger.py

```diff
@@ -19,3 +19,21 @@ def get_logger(name, level=logging.DEBUG, stdout=False):
         logger.addHandler(handler)
 
     return logger
+
+
+def get_custom_logger(name, log_file: str, level=logging.DEBUG, stdout=False):
```

Comment:
I would not make separate log files for each module, it's easier to spot the order of events in one file. I'd rather have separate log files for API and for CLI and make API logs less noisy (like removing messages on offset change call etc.)